src/main/webapp/js/components/fields/visit-schedule-fields.jsx

```diff
@@ -1,15 +1,59 @@
-import React from 'react';
-
-const VisitScheduleFields = () => (
-  <div className="container-fluid">
-    <h1>
-      VisitScheduleFields VisitScheduleFields VisitScheduleFields
-      VisitScheduleFields
-      <br />
-      Placeholder Placeholder Placeholder Placeholder
-      <br />
-    </h1>
-  </div>
-);
-
-export default VisitScheduleFields;
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+
+import FieldConfigPage from './field-config-page';
+import { fetchFieldConfig, saveFieldConfig, deleteFieldConfig } from '../../actions/index';
+
+import {
+  FETCH_VISIT_CONFIG,
+  SAVE_VISIT_CONFIG,
+  DELETE_VISIT_CONFIG,
+} from '../../actions/types';
+
+const VISIT = 'VISIT';
+
+class VisitScheduleFields extends Component {
```

Comment:
this should be named VisitFields, not VisitScheduleFields, because you're changing here fields in the Visit, not just the schedule page
I think there might be 2 separate pages, one for editing the visit entity and one with the visit schedule